src/main/java/org/commcare/formplayer/aspects/MetricsAspect.java

```diff
@@ -30,6 +34,9 @@
 @Order(1)
 public class MetricsAspect {
 
+    private static final String EXTREMELY_SLOW_REQUEST = "long_request"; // artifact of prior naming
```

Comment:
These strings (and variable names) are hard to tell apart, it would be nice if the difference was more meaningful.